### PR TITLE
Fix iOS builds

### PR DESCRIPTION
**Pull request by kried**
Fixes failing iOS CI builds

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 version: 2.1
 
+orbs:
+  macos: circleci/macos@2.4.1
+  gh: circleci/github-cli@2.0
```
Comment (volkdmitri) [inline .circleci/config.yml:5]: Looks like `gh` is unused.

**Comment by volkdmitri** [comment]
Could you please add a bit of details on why it fixes the issue?

File: .circleci/config.yml (hunk 1 above)
Comment (kried) [reply .circleci/config.yml:5]: It is used inside of the `Scripts/update-version.sh` script.

**Comment by kried** [comment]
> Could you please add a bit of details on why it fixes the issue?
@volkdmitri 

The issue: all CI build in this repo fail due to usage of [deprecated](https://circleci.com/docs/using-macos/#available-resource-classes) `resource_class: macos.x86.medium.gen2`.
The fix is to switch to available `resource_class` usage, but this will require configuring additional tools, like [rosetta orb](https://support.circleci.com/hc/en-us/articles/17402023738651-Installing-Rosetta-on-Apple-Silicon-M1-executors) and `gh orb`

**Review by volkdmitri — APPROVED** [review]

**Comment by volkdmitri** [comment]
> > Could you please add a bit of details on why it fixes the issue?
> > @volkdmitri
> 
> The issue: all CI build in this repo fail due to usage of [deprecated](https://circleci.com/docs/using-macos/#available-resource-classes) `resource_class: macos.x86.medium.gen2`. The fix is to switch to available `resource_class` usage, but this will require configuring additional tools, like [rosetta orb](https://support.circleci.com/hc/en-us/articles/17402023738651-Installing-Rosetta-on-Apple-Silicon-M1-executors) and `gh orb`

Thank you.

File: .circleci/config.yml (hunk 1 above)
Comment (volkdmitri) [reply .circleci/config.yml:5]: Thanks.

**Pull request merged**